requirements.txt

```diff
@@ -0,0 +1,2 @@
+mkdocs
```

Comment:
For longevity of this repository, shouldn't specific versions be forced? 
At least force so no major version change can happen.

Reply:
Good point. Updated.